public/data/krshdev.json

```
@@ -1,41 +1,83 @@
 {
   "name": "Krish Gupta",
   "type": "personal",
-  "bio": "An enthusiast student developer looking forward to contribute to open source!",
+  "bio": "An enthusiastic and ambitious full stack developer. Specialising in Web Development, JavaScript and GitHub",
   "avatar": "https://github.com/krshdev.png",
   "links": [
     {
-      "name": "Follow me on GitHub",
+      "name": "Collaborative Work on GitHub",
       "url": "https://github.com/krshdev",
       "icon": "github"
     },
     {
-      "name": "Follow me on Twitter",
+      "name": "Programming Content on Twitter",
       "url": "https://twitter.com/krshdev",
       "icon": "twitter"
     },
     {
-      "name": "Checkout my website",
-      "url": "https://krshdev.co",
-      "icon": "globe"
+      "name": "Blogs & Articles about Development",
+      "url": "https://blog.krshdev.co",
+      "icon": "book"
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://linkedin.com/in/krshdev",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Check out my instagram",
```

You might want to capitalize the `i` here. What do you think?
```suggestion
      "name": "Check out my Instagram",
```